app/frontend/wallet/shelley/shelley-transaction-planner.ts

```diff
@@ -1,10 +1,10 @@
 import {encode} from 'borc'
-import {base58} from 'cardano-crypto.js'
+import bech32 from './helpers/bech32'
 
-import {TxInputFromUtxo} from './byron-transaction'
+import {ShelleyTxInputFromUtxo} from './shelley-transaction'
 
 import {TX_WITNESS_SIZE_BYTES} from '../constants'
-import CborIndefiniteLengthArray from './helpers/CborIndefiniteLengthArray'
+import CborIndefiniteLengthArray from '../byron/helpers/CborIndefiniteLengthArray'
```

Comment:
this is also a shelley helper (as opposed to ITN) to we should move it to some generic helper directory